<!-- -*- mode: XML -*- -->

<launch>

  # Publish Velodyne lidar scans
  <node name="converter_velodyne_node" pkg="converter_velodyne" type="converter_velodyne_node"
    required="true" output="screen">
    <rosparam file="$(find converter_velodyne)/config/converter_velodyne.yaml" command="load" />
    <remap from="velodyne_packets" to="/velodyne/rear/packets"/>
  </node>

</launch>